coretex/cli/modules/node.py

```diff
@@ -236,6 +236,23 @@ def selectNodeMode(storagePath: str) -> Tuple[int, Optional[int]]:
     return availableNodeModes[selectedMode], None
 
 
+def promptRam(config: Dict[str, Any], ramLimit: int, retryCount: int = 0) -> int:
+    nodeRam: int = clickPrompt("Node RAM memory limit in GB (press enter to use default)", config_defaults.DEFAULT_RAM_MEMORY, type = int)
+
+    if ramLimit < config_defaults.MINIMUM_RAM_MEMORY:
+        raise RuntimeError(f"Minimum Node RAM requirement ({ramLimit}GB) is higher than your current Docker desktop RAM limit ({ramLimit}GB). Please adjust resource limitations in Docker Desktop settings to match Node requirements.")
+
+    if nodeRam > ramLimit:
+        errorEcho(f"WARNING: RAM limit in Docker Desktop ({ramLimit}GB) is lower than the configured value ({config['ramLimit']}GB). Please adjust resource limitations in Docker Desktop settings.")
```

Comment:
This is an error not a warning. When a warning happens it means that the situation which is a result of it does not affect the normal flow of the program, whilst when an error happens it means that the normal flow gets interrupted.